src/commands/explore.rs

```diff
@@ -50,7 +55,35 @@ impl ExploreCommand {
             .with_context(|| format!("failed to create file: {}", output.display()))?;
         let mut output_file = std::io::BufWriter::new(output_file);
 
-        wasmtime_explorer::generate(&config, self.target.as_deref(), &bytes, &mut output_file)?;
+        let clif_dir = output
+            .parent()
+            .map::<Result<PathBuf>, _>(|output_dir| {
+                let clif_dir = output_dir.join("clif");
+                if let Err(err) = create_dir(&clif_dir) {
+                    match err.kind() {
+                        io::ErrorKind::AlreadyExists => {
+                            return Err(anyhow!("clif directory exists"));
+                        }
```

Comment:
I think we can just overwrite the existing files, otherwise doing `wasmtime explore` in the same location twice will be really annoying, since we'd have to remove a bunch of stuff before we could run the command again.

Reply:
And for what it's worth, we overwrite the html file.

Reply:
Oh, actually, let's just use a [temp directory](https://docs.rs/tempfile/latest/tempfile/fn.tempdir.html) for this. We already use the `tempfile` crate as a dev-dependency for the `wasmtime` crate, so I think we can also add that as a normal, but optional, dependency and then turn the dependency on when the `explore` cargo feature is enabled.

Reply:
> I think we can just overwrite the existing files, otherwise doing `wasmtime explore` in the same location twice will be really annoying, since we'd have to remove a bunch of stuff before we could run the command again.

Actually, it removes the directory afterwards, that's why I thought it was unsafe to delete an existing directory. I will change to use the temp directory